app/routes/surveys/AddSurveyRoute.js

```diff
@@ -37,5 +91,10 @@ const mapDispatchToProps = {
 
 export default compose(
   replaceUnlessLoggedIn(LoginPage),
+  prepare(loadData, [
+    'notFetching',
```

Comment:
why do you need `notFetching` in here?

Reply:
I suppose that should be in a `loadingIndicator`